cli/formatters/pollingprofile.go

```diff
@@ -0,0 +1,66 @@
+package formatters
+
```

Comment:
We need to refactor the way we handle formatters in the future

Reply:
+1. This is a very hardcoded approach and won't scale to match the dynamic nature of the resource manager